### ISSUE TITLE
Elevated plasma propionylcarnitine, C3:0

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#5959]
**Preferred term label:** Elevated plasma propionylcarnitine, C3:0

**Synonyms**

**Definition (free text, please give PubMed ID)** Elevated propionylcarnitine concentration in the blood., PMID: 24707392

**Parent term (use hpo.jax.org/app)** Abnormal circulating fatty-acid concentration HP:0004359

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
